sdk/src/main/java/io/dapr/client/DaprClient.java

```diff
@@ -662,6 +665,15 @@ Flux<SubscribeConfigurationResponse> subscribeConfiguration(String storeName, Li
    * @return Mono of {@link UnsubscribeConfigurationResponse} instance.
    */
   Mono<UnsubscribeConfigurationResponse> unsubscribeConfiguration(UnsubscribeConfigurationRequest request);
+  
+  /**
+   * ScheduleJobAlpha1 creates and schedules a job.
+   * 
+   * @param <T> The type of the data for the job.
+   * @param job job to be scheduled
+   * @return a Mono plan of type Void.
+   */
+  <T extends Message> Mono<Void> scheduleJobAlpha1(Job<T> job);
```

Comment:
It does not need to be a protobuf message. Although the implementation can be smart enough to handle Message as a special case and not use the serializer.

This interface is agnostic of gRPC protocol.

Reply:
@artursouza I can't convert generic type T to Any data type. It is expecting generic type to extend Message

Reply:
@charan2628 you are defining the method signature, what @artursouza is suggesting is not to use Message as the public return type.

Reply:
@artursouza and @salaboy  got it I am using DaprObjectSerializer